src/test/java/com/example/petbuddybackend/service/animal/AnimalServiceIntegrationTest.java

```diff
@@ -52,16 +53,36 @@ void getAnimal_WhenAnimalDoesNotExists_ThenThrowNotFoundException() {
 
     @Test
     void getAnimalAttributes() {
-        Set<AnimalAttribute> animalAttributes = animalService.getAnimalAttributes(List.of(1L, 2L));
+        Set<AnimalAttribute> animalAttributes = animalService.getAnimalAttributesOfAnimal(List.of(1L, 2L));
         assertNotNull(animalAttributes);
         assertEquals(2, animalAttributes.size());
     }
 
     @Test
     void getAnimalAttributes_WhenDuplicated_ThenFlatten() {
-        Set<AnimalAttribute> animalAttributes = animalService.getAnimalAttributes(List.of(1L, 1L));
+        Set<AnimalAttribute> animalAttributes = animalService.getAnimalAttributesOfAnimal(List.of(1L, 1L));
         assertNotNull(animalAttributes);
         assertEquals(1, animalAttributes.size());
     }
 
+    @Test
+    void getAnimalAttributesOfAnimal() {
+        Map<String, List<String>> result = animalService.getAnimalAttributesOfAnimal("DOG");
+        assertNotNull(result);
```

Comment:
```suggestion
        assertNotNull(result);
        assertFalse(amenities.isEmpty());
```

Tu też bym tego asserta dał

Reply:
Nie dawałem assertow na ilość bo to się będzie zmieniało wraz z dodawaniem do bazy